src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+$cell-width: 100px;
+$gap: 1px;
+
+@each $day, $index in $days-of-week {
+  .calendar--start-day-#{$day} .calendar__day:nth-child(1) {
+    margin-left: calc(($cell-width + $gap) * $index);
+  }
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-style: normal;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 714px;
+  box-sizing: border-box;
+  margin: 80px 160 0;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should use units for all property values where they are allowed. Here, the `margin` shorthand property is missing the unit for the second value (160). It should be `160px`.